corehq/apps/api/resources/messaging_event/utils.py

```diff
@@ -0,0 +1,29 @@
+from tastypie.exceptions import BadRequest
+
+
+def get_limit_offset(param_name, request_data, default, max_value=None):
+    value = request_data.get(param_name, default)
+    try:
+        value = int(value)
+    except ValueError:
+        raise BadRequest(f"Invalid {param_name} '{value}' provided. Please provide a positive integer.")
```

Comment:
Do we translate API errors? They're end-user-facing.

Reply:
They are end-user facing, I'll add translation markers

Reply:
639ea763cad